File: algorithms/collatz-sequence/javascript/collatz.js
let collatz = (n) => {
  if (n < 1) {
    console.log('Enter a positive prime number!')
    return null
  }
  while (n != 1) {
    if (!(n % 2)) {
      n /= 2
    } else {
      n = 3 * n + 1
    }
    console.log(n)
  }
  console.log(n)

}

File: algorithms/collatz-sequence/python/collatz.py
def collatz (n):
  if (n < 1):
    print('Enter a positive integer!')
    return None
  else:
    while (n != 1):
      if (not n % 2):
        n /= 2
      else:
        n = 3 * n + 1
      print(int(n))
  print(int(n))
